guides/doc-Release_Notes/topics/katello-4.13.0.adoc

```diff
@@ -0,0 +1,136 @@
+= Katello 4.13.0
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1788[Redmine]
+
+== Katello
+
+* pass:[Package rubygem-dynflow not listed in a list of packages] - https://projects.theforeman.org/issues/37457[#37457]
+* pass:[Cannot update packages on non-EL hosts] - https://projects.theforeman.org/issues/37340[#37340]
+* pass:[Fix upstream lint issues] - https://projects.theforeman.org/issues/37331[#37331]
+* pass:[It is possible to end up with the wrong remote type (uln vs. normal) for yum content] - https://projects.theforeman.org/issues/37279[#37279]
+* pass:[Default Organization View is not listed first on the CV select screen in Change Content Source] - https://projects.theforeman.org/issues/37229[#37229]
+* pass:[It should be possible to upload a package / repos profile from UI] - https://projects.theforeman.org/issues/37191[#37191]
+* pass:[content_view_components is not preloaded in content_view controller] - https://projects.theforeman.org/issues/37108[#37108]
+
+=== API
+
+* pass:[--content-view-filter-id only works for ID-type filters] - https://projects.theforeman.org/issues/37394[#37394]
+* pass:[API endpoint for activation_keys/:id/product_content should be TRUE by default] - https://projects.theforeman.org/issues/37350[#37350]
+
+=== Activation Key
+
+* pass:[Change the default setting for "Limit to environment" on the activationkey and content host pages to true] - https://projects.theforeman.org/issues/37214[#37214]
+
+=== Alternate Content Sources
+
+* pass:[Fix ACS randomly failing VCR tests] - https://projects.theforeman.org/issues/37277[#37277]
+
+=== Container
+
+* pass:[Allow pushing container images to Pulp without indexing] - https://projects.theforeman.org/issues/37302[#37302]
+
+=== Content Credentials
+
+* pass:[asterisk symbol is missing for required field] - https://projects.theforeman.org/issues/37482[#37482]
+
+=== Content Views
+
+* pass:[Content view publish failing with katello_repository_rpms_id_seq reached maximum value error] - https://projects.theforeman.org/issues/37403[#37403]
+* pass:[Content view repositories link points to broken link on composite view UI] - https://projects.theforeman.org/issues/37269[#37269]
+* pass:[Newly imported content views show as needs publish] - https://projects.theforeman.org/issues/37254[#37254]
+* pass:[Allow repairing content view versions] - https://projects.theforeman.org/issues/37237[#37237]
+* pass:[[RFE] Block content view publishing during repository publication tasks] - https://projects.theforeman.org/issues/37139[#37139]
```

Comment:
We could add this as a headline feature :

Katello will now prevent users when publishing a content view if a child repository is syncing and vice-versa to avoid any data integrity issues.

Reply:
updated